package.json

```diff
@@ -4,10 +4,21 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
+    "clean:reports": "rm -rf cypress/reports && mkdir cypress/reports",
+    "clean:reportsWindows": "rmdir /S /Q cypress\\reports && mkdir cypress\\reports",
     "test": "cypress run --headless --browser chrome",
+    "testWindows": "cypress run --headless --browser chrome",
```

Comment:
if you use the git bash console in windows you will not need this command

Reply:
Tested with git bash after removing windows command. worked as expected. thanks @holgiosalos